### PR TITLE
Add AOCODAF4V3-SDCard

### PR DESCRIPTION
**Pull request by 634296207**
Add AOCODAF4V3-SDCard

### CODE REVIEW

File: src/main/target/AOCODARCF4V3/CMakeLists.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
-target_stm32f405xg(AOCODARCF4V3)
+target_stm32f405xg(AOCODARCF4V3_SDCard)
+target_stm32f405xg(AOCODARCF4V3_Flash)
```
Comment (lida2003) [inline src/main/target/AOCODARCF4V3/CMakeLists.txt]: Please use AOCODARCF4V3 for default (flash version)

We have company naming rule, default for flash version, which is:
- AOCODARCF4V3   //Flash version
- AOCODARCF4V3_SD  // SDCard version

Then we need to handle AOCODARCF4V3_SD macron only.,

File: src/main/target/AOCODARCF4V3/target.h [hunk 2]
```diff
@@ -19,8 +19,13 @@
...
 #define USE_TARGET_CONFIG
 
-#define TARGET_BOARD_IDENTIFIER         "AOF4V3"
-#define USBD_PRODUCT_STRING             "AocodaRCF4V3"
+#ifdef AOCODARCF4V3_Flash
+#define TARGET_BOARD_IDENTIFIER "AOF4V3F"
+#define USBD_PRODUCT_STRING     "AocodaRCF4V3_Flash"
+#else
+#define TARGET_BOARD_IDENTIFIER "AOF4V3SD"
+#define USBD_PRODUCT_STRING     "AOCODARCF4V3_SDCard"
```
Comment (lida2003) [inline src/main/target/AOCODARCF4V3/target.h]: Same issue, please consult company product naming rule.

**Review by lida2003 — CHANGES_REQUESTED** [review]

File: src/main/target/AOCODARCF4V3/target.h (hunk 2 above)
Comment (634296207) [reply src/main/target/AOCODARCF4V3/target.h]: Okay, I've already changed it

Comment (lida2003) [reply src/main/target/AOCODARCF4V3/target.h]: > Okay, I've already changed it

No, you didn't understand what I mean. Check the above comments.

> We have company naming rule, default for flash version, which is:
> 
>     AOCODARCF4V3 //Flash version
>     AOCODARCF4V3_SD // SDCard version
> 
> Then we need to handle AOCODARCF4V3_SD macron only.,

File: src/main/target/AOCODARCF4V3/target.h [hunk 3]
```diff
@@ -19,8 +19,13 @@
 
 #define USE_TARGET_CONFIG
 
-#define TARGET_BOARD_IDENTIFIER         "AOF4V3"
-#define USBD_PRODUCT_STRING             "AocodaRCF4V3"
+#ifdef AOCODARCF4V3
+#define TARGET_BOARD_IDENTIFIER "AOF4V3F"
```
Comment (lida2003) [inline src/main/target/AOCODARCF4V3/target.h]: Just AOF4V3 for flash version, AOF4V3SD for SDCard version.

**Review by lida2003 — CHANGES_REQUESTED** [review]

File: src/main/target/AOCODARCF4V3/target.h (hunk 3 above)
Comment (634296207) [reply src/main/target/AOCODARCF4V3/target.h]: Change AOF4V3F to AOF4V3

**Comment by 634296207** [comment]
c14c00821bf9d6dd72ee423c8f416c6

PR self-test passed
c14c00821bf9d6dd72ee423c8f416c6

PR self-test passed

**Comment by lida2003** [comment]
> c14c00821bf9d6dd72ee423c8f416c6
> 
> PR self-test passed c14c00821bf9d6dd72ee423c8f416c6
> 
> PR self-test passed

All PR related functions has passed, or just passed compiling and linking?

**Comment by 634296207** [comment]
F405V3-SD V3(HW) self test passed.

**Review by lida2003 — APPROVED** [review]
LGTM

**Pull request merged**